static/ejs/checkin.js

```diff
@@ -7,15 +7,17 @@ function print() { __p += __j.call(arguments, '') }
 with (obj) {
 __p += '<li data-team-id="' +
 ((__t = ( checkin.team.id )) == null ? '' : __t) +
-'">\n  <span class="smiley">☺ </span>\n  <span class="team-name">' +
+'">\r\n  <span class="smiley">☺ </span>\r\n  <span class="team-id">' +
```

Comment:
Since we're touching this, we should consider changing the embedded smiley into a [unicode character](https://www.unicode.org/emoji/charts/full-emoji-list.html) for the proper use.  Thoughts?

Reply:
Sounds good, I can take a look at that. Any specific smiley you want added?

Reply:
Have fun with it!  Mind opening a new PR with that small change?